### PR TITLE
Rb 4.1.x bluemsx colecovision

### PR DESCRIPTION
**Pull request by rockaddicted**
Please make sure your PR is ready to be merged !
- [x] You added the changes in CHANGELOG.md
- [x] You choose the right repository branch to make the PR
- [x] You described the PR as below

Fixes #300

Changes :
- added MSX2 platform (by default) to bluemsx (some games will be now displayed in english instead of japanese)
-  added, on MSX,  F1/F2/F5 keys on joystick (needed by some games)
- added ColecoVision support to bluemsx libretro core
- added controller support to ColecoVision

Related to (put here the others PR in other repositories)

### CODE REVIEW

File: board/recalbox/fsoverlay/recalbox/share_init/roms/colecovision/_readme.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## RECALBOX - SYSTEM COLECOVISION ##
+
+Put your msx roms in this directory.
+
+Rom files must have a ".col" extension.
+
+You can download msx1/2 roms on planetemu :
```
Comment (paradadf) [inline board/recalbox/fsoverlay/recalbox/share_init/roms/colecovision/_readme.txt]: colecovision

Comment (rockaddicted) [inline board/recalbox/fsoverlay/recalbox/share_init/roms/colecovision/_readme.txt]: Haha missed a copy paste
I'll correct it, thx

File: board/recalbox/fsoverlay/recalbox/share_init/roms/colecovision/_readme.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## RECALBOX - SYSTEM COLECOVISION ##
+
+Put your colecovision roms in this directory.
+
+Rom files must have a ".col" extension.
+
+You can download msx1/2 roms on planetemu :
```
Comment (paradadf) [inline board/recalbox/fsoverlay/recalbox/share_init/roms/colecovision/_readme.txt]: One more here ;)

**Comment by bigboo3000** [comment]
Are you planning to upstream this patch to libretro?

**Comment by digitalLumberjack** [comment]
Nice :)

**Pull request merged**